tests/suites/os/tests/config-json/index.js

```diff
@@ -170,6 +170,12 @@ module.exports = {
 					'inactive',
 					context.link
 				);
+				const dnsmasqInvocationId = await context.worker.executeCommandInHostOS(
+					[
+						`systemctl`, `show`, `-p`, `InvocationID`, `--value`, `dnsmasq.service`
```

Comment:
Do we need this check surrounding lines 145-153 and lines 233-241? Why is this only needed for setting dnsServers to null and not the other two tests?

Reply:
@klutchell At 145, and 233 it looks like we're waiting for dnsmasq to be running with the new settings, so it's ostensibly not needed in these cases. The difference with the test at 207 is that we make the change and check to see if the settings are applied before they've been made.